sdk/src/main/java/com/google/cloud/dataflow/sdk/io/OffsetBasedSource.java

```diff
@@ -264,7 +264,13 @@ public final boolean start() throws IOException {
 
     @Override
     public final boolean advance() throws IOException {
-      return advanceImpl() && rangeTracker.tryReturnRecordAt(isAtSplitPoint(), getCurrentOffset());
```

Comment:
Is there somewhere to document the change in "pattern"? I remember seeing the above pattern as the "suggested" way to do readers.